Dockerfile

```diff
@@ -92,6 +94,7 @@ RUN apt-get update \
 
 RUN ln -s /bin/tar /bin/gtar \ 
     && R --silent -e "devtools::install_github('earthlab/qtoolkit', dependencies = FALSE)" 
+    && R --silent -e "devtools::install_github('oswaldosantos/ggsn', dependencies = FALSE)" 
```

Comment:
@jvtcl did you try to build this locally. i'm getting a docker build error. i believe it needs a `\` to parse a new line. i'll add this directly now but please be sure to do that for the future!!